flake.nix

```diff
@@ -83,13 +83,15 @@
         formatter = pkgs.nixpkgs-fmt;
 
         devShells.default = pkgs.mkShell {
+          # include dependencies of the default package
+          inputsFrom = [ self.packages.${localSystem}.default ];
```

Comment:
.. Wasn't this the thing we _didn't_ want to do?

Reply:
It was loading all the checks before. This is a bit more sensible.

Reply:
Ah, right. inputsFrom mentioned checks earlier, I remember

Reply:
So basically this time it doesn't include our crates themselves, but just their dependencies?

Reply:
(since our package itself was part of checks which was used previously)

Reply:
Exactly!